tfhe/src/js_on_wasm_api/mod.rs

```diff
@@ -1,15 +1,15 @@
 #[cfg(feature = "shortint-client-js-wasm-api")]
-pub mod shortint;
+mod shortint;
 #[cfg(feature = "shortint-client-js-wasm-api")]
 pub use shortint::*;
 
 #[cfg(feature = "boolean-client-js-wasm-api")]
-pub mod boolean;
+mod boolean;
 #[cfg(feature = "boolean-client-js-wasm-api")]
 pub use boolean::*;
 
 #[cfg(feature = "parallel-wasm-api")]
 pub use wasm_bindgen_rayon::init_thread_pool;
 
-pub mod high_level_api;
-pub use high_level_api::*;
```

Comment:
Why was it renamed ?

Reply:
I took it from Arthur PR https://github.com/zama-ai/tfhe-rs/pull/470
It's already been merged in https://github.com/zama-ai/tfhe-rs/pull/494
I need to rebase this PR

Reply:
Ok